.github/workflows/wren-launcher-ci.yaml

```
@@ -0,0 +1,141 @@
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: wren-launcher
+
+jobs:
+  lint-and-test:
+    # run this job only if the PR is labeled with "launcher" or if it's a push to main
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'launcher') || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)